tests/pipeline/test_pipeline_algo.py

```diff
@@ -512,7 +513,7 @@ def initialize(context):
             attach_pipeline(pipeline, 'test')
 
         def handle_data(context, data):
-            today = get_datetime()
+            today = normalize_date(get_datetime())
```

Comment:
Does this mean that existing pipeline algos will change results without normalizing the date?